site.ts

```diff
@@ -329,22 +356,72 @@ export function inject({ config, posthog }) {
         return formElement
     }
 
+    const createMultipleChoicePopup = (survey) => {
+        const surveyQuestion = survey.questions[0].question
+        const surveyDescription = survey.questions[0].description
+        const surveyQuestionChoices = survey.questions[0].choices
+        const singleOrMultiSelect = survey.questions[0].type
+        const form = `
+        <div class="survey-${survey.id}-box">
+            <div class="cancel-btn-wrapper">
+                <button class="form-cancel" type="cancel">${cancelSVG}</button>
+            </div>
+            <div class="survey-question">${surveyQuestion}</div>
+            ${surveyDescription ? `<span class="description">${surveyDescription}</span>` : ''}
+            <div class="multiple-choice-options">
+            ${surveyQuestionChoices.map((option, idx) => {
+                const inputType = singleOrMultiSelect === 'multiple_single' ? 'radio' : 'checkbox'
+                const singleOrMultiSelectString = `<div class="choice-option"><input type=${inputType} id=surveyQuestionMultipleChoice${idx} name="choice" value="${option}">
+                <label for=surveyQuestionMultipleChoice${idx}>${option}</label></div>`
+            return singleOrMultiSelectString
+        }).join(' ')}
+            </div>
+            <div class="bottom-section">
+            <div class="buttons">
+                <button class="form-submit" type="submit">${survey.appearance?.submitButtonText || 'Submit'}</button>
+            </div>
+            <div class="footer-branding"><div>powered by ${posthogLogo} PostHog</div></div>
+        </div>
+
+        </div>
+        `
+        const formElement = Object.assign(document.createElement('form'), {
+            className: `survey-${survey.id}-form`,
+            innerHTML: form,
+            onsubmit: (e) => {
+                e.preventDefault()
+                const selectedChoices = singleOrMultiSelect === 'multiple_single' ? e.target.querySelector('input[type=radio]:checked').value : [...e.target.querySelectorAll('input[type=checkbox]:checked')].map((choice) => choice.value)
+                posthog.capture('survey sent', {
+                    $survey_name: survey.name,
+                    $survey_id: survey.id,
+                    $survey_question: survey.question,
+                    $survey_response: selectedChoices,
```

Comment:
@neilkakkar Could use some feedback here on how I should return multiple choice options + values. I'm using a basic array for choice options but I was wondering if it would make sense to assign an index to them and as objects instead? That way users can choose how they want to order their options and more (idk what more right now, but possibly logic based option selection) 

Current:
```
{ type: 'multiple_single', 
question: "which of these options will you choose?", 
choices: ["choice 1", "choice 2", "choice 3"] }
```

Alternative:
```
{ type: 'multiple_single', 
question: "which of these options will you choose?", 
choices: [ { index: 1, text: "choice 1" }, { index: 2, text: "choice 2"} ]
```